extension/InspectionDoc/src/Resources/assets/inspection_docs.json

```diff
@@ -0,0 +1 @@
al","link":"/docs/sdk/dev/development-tools/sniffs/spryker/php/not-equal-sniff.html"},{"inspectionId":"Spryker.PHP.PhpSapiConstant","link":"/docs/sdk/dev/development-tools/sniffs/spryker/php/php-sapi-constant-sniff.html"},{"inspectionId":"Spryker.PHP.PreferCastOverFunction","link":"/docs/sdk/dev/development-tools/sniffs/spryker/php/prefer-cast-over-function-sniff.html"},{"inspectionId":"Spryker.PHP.RemoveFunctionAlias","link":"/docs/sdk/dev/development-tools/sniffs/spryker/php/remove-function-alias-sniff.html"},{"inspectionId":"Spryker.PHP.ShortCast","link":"/docs/sdk/dev/development-tools/sniffs/spryker/php/short-cast-sniff.html"},{"inspectionId":"Spryker.PHP.SingleQuote","link":"/docs/sdk/dev/development-tools/sniffs/spryker/php/single-quote-sniff.html"},{"inspectionId":"Spryker.Testing.AssertPrimitives","link":"/docs/sdk/dev/development-tools/sniffs/spryker/testing/assert-primitives-sniff.html"},{"inspectionId":"Spryker.Testing.ExpectException","link":"/docs/sdk/dev/development-tools/sniffs/spryker/testing/expect-exception-sniff.html"},{"inspectionId":"Spryker.Testing.Mock","link":"/docs/sdk/dev/development-tools/sniffs/spryker/testing/mock-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.CommaSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/comma-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.ConcatenationSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/concatenation-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.DocBlockSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/doc-block-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.EmptyEnclosingLine","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/empty-enclosing-line-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.EmptyLines","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/empty-lines-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.FunctionSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/function-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.ImplicitCastSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/implicit-cast-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.MemberVarSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/member-var-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.MethodSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/method-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.NamespaceSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/namespace-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.ObjectAttributeSpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/object-attribute-spacing-sniff.html"},{"inspectionId":"Spryker.WhiteSpace.TernarySpacing","link":"/docs/sdk/dev/development-tools/sniffs/spryker/white-space/ternary-spacing-sniff.html"}]
```

Comment:
must be an empty file. Any content generated here must be only via PR provided by the Tech Writer side.